sequences/__init__.py

```diff
@@ -3,13 +3,15 @@
 
 def get_next_value(
         sequence_name='default', initial_value=1,
-        *, nowait=False, using=None):
+        *, nowait=False, using=None, max_value=None):
     """
     Return the next value for a given sequence.
 
     """
     # Inner import because models cannot be imported before their application.
     from .models import Sequence
+    if max_value:
+        assert initial_value < max_value
```

Comment:
With `initial_value = 1` and `max_value = 0`, this assertion doesn't raise an error, while it should.